flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -156,9 +158,9 @@ public class YarnClusterDescriptor implements ClusterDescriptor<ApplicationId> {
     private final boolean sharedYarnClient;
 
     /** Lazily initialized list of files to ship. */
-    private final List<File> shipFiles = new LinkedList<>();
+    private final List<Path> shipFiles = new LinkedList<>();
 
-    private final List<File> shipArchives = new LinkedList<>();
+    private final List<Path> shipArchives = new LinkedList<>();
```

Comment:
How about adding a comment to describe we have converted the option path str to the Path with schema and absolute path? It's more clear for other developers, and it's clear to use them.

Reply:
@1996fanrui updated.